src/Pos/Script/Examples.hs

```diff
@@ -139,7 +139,7 @@ multisigRedeemer txSigData sks = fromE $ parseRedeemer Nothing [text|
     mkCons Nothing s = sformat ("(Cons Nothing "%build%")") s
     mkCons (Just sig) s = sformat
         ("(Cons (Just #"%fullSignatureHexF%") "%build%")") sig s
-    shownSigs = foldr mkCons "Nil" (map (fmap (`sign` txSigData)) sks)
+    shownSigs = foldr mkCons "Nil" (map (fmap (`safeSign` txSigData)) sks)
```

Comment:
We have `<<$>>` in `universum` for `map . fmap`: 

https://github.com/serokell/universum/blob/d4199cb7e67265a8b2ea1e5c0d9fa3142e473ab5/src/Functor.hs#L17